### DISCUSSION TITLE
JS Version

### DISCUSSION THREAD

**Discussion by codecompanion-ai** [crewAIInc/crewAI#1110] [Feature Requests]
Please also make JS version

**Comment by joaomdmoura** [comment]
yup! in the roadmap! 😎👉👉

**Comment by arash-decidr** [comment]
@joaomdmoura, approximately when?

**Comment by rohan-uiuc** [comment]
Hi @joaomdmoura is there a timeline for this? I'm looking forward to use crewai in my js project. Thanks!
